src/main/java/org/damiane/exception/UnauthorizedAccessException.java

```diff
@@ -5,4 +5,3 @@ public UnauthorizedAccessException(String message) {
         super(message);
```

Comment:
Mind using Exception instead

Reply:
Changed from RuntimeException to Exception